content/docs/start/bitbucket.md

```diff
@@ -0,0 +1,71 @@
+# Get Started with CML on Bitbucket
+
+Here, we'll walk through a tutorial to start using CML. For simplicity, we'll
+show the demo in Bitbucket Pipelines, but instructions are pretty similar for
+all the supported CI systems.
+
+1. Fork our
+   [example project repository](https://bitbucket.org/iterative-ai/example-cml).
+
+   ![](/img/bitbucket_fork_cml_project.png)
+
+   The following steps can all be done in the Bitbucket browser interface.
+   However, to follow along the commands, we recommend cloning your fork to your
+   local workstation:
+
+   ```bash
+   git clone https://bitbucket.org/<your-username>/example-cml
+   ```
+
+2. To create a CML workflow, copy the following into a new file,
+   `bitbucket-pipelines.yml`:
+
+   ```yaml
+   image: iterativeai/cml:0-dvc2-base1
+   pipelines:
+     default:
+       - step:
+           name: Train model
+           script:
+             - pip install -r requirements.txt
+             - python train.py
+
+             - cat metrics.txt > report.md
+             - cml-publish confusion_matrix.png --md >> report.md
+             - cml-send-comment report.md
+   ```
+
+3. In your text editor of choice, edit line 16 of `train.py` to `depth = 12`.
```

Comment:
```suggestion
3. In your text editor of choice, edit line 15 of `train.py` to `depth = 12`.
```

Reply:
[It's line 16 on the repository](https://bitbucket.org/iterative-ai/cml-base-case/src/32e60c3e0cc124e40b1681b9b7b0458f3a238474/train.py#lines-16) 🤔